src/workerd/server/workerd-api.h

```diff
@@ -285,4 +287,29 @@ class WorkerdApi final: public Worker::Api {
 kj::Maybe<jsg::Bundle::Reader> fetchPyodideBundle(
     const api::pyodide::PythonConfig& pyConfig, kj::StringPtr version);
 
+// Downloads a package with retry logic (up to 3 attempts with 5-second delays)
```

Comment:
workerd-api.h is already fairly complex. Is it possible to move these new APIs into a separate pyodide compilation unit or maybe into the existing `api/pyodide.h/c++` ?